google/cloud/spanner/row.cc

```diff
@@ -51,6 +52,7 @@ Row::Row(std::vector<Value> values,
   if (values_.size() != columns_->size()) {
     GCP_LOG(FATAL) << "Row's value and column sizes do not match: "
                    << values_.size() << " vs " << columns_->size();
+    std::exit(EXIT_FAILURE);
```

Comment:
Why the call to `std::exit`?  Presumably we want that to happen in the `GCP_LOG(FATAL)` line?